Regression_CF.Rmd

```diff
@@ -0,0 +1,108 @@
+---
+title: "Making My Own Functions: Check Regression Line- Comparison of Petal and Sepal Lengths / Widths"
+author: "Claire"
+date: "2020-01-22"
+output: pdf_document
+---
+
+We read in the desired dataset (preloaded to R):
+```{r}
+read.csv("iris.csv")
+```
+
+We then defined objects:
+```{r}
+x <- Slength
+y <- Plength
```

Comment:
Donʻt forget to test your script after clearing your workspace!!

processing file: Regression_CF.Rmd
  |..                                                  |   5%
  |.....                                               |   9%
  |.......                                             |  14%
  |.........                                           |  18%
label: unnamed-chunk-2
Quitting from lines 15-22 (Regression_CF.Rmd) 
Error in eval(expr, envir, enclos) : object 'Slength' not found